Gemfile.lock

```diff
@@ -331,7 +331,7 @@ GEM
       win32-ipc (>= 0.6.0)
     win32-process (0.8.3)
       ffi (>= 1.0.0)
-    win32-service (1.0.1)
+    win32-service (2.1.2)
```

Comment:
We should bump the required version in the gemspec to what we need for this new functionality. That way we a gem install doesn't run with win32-service 1.x and fail

Reply:
great catch. I just updated and re-pushed.